.github/workflows/composer-validate.yml

```diff
@@ -0,0 +1,35 @@
+name: "Check validity of composer.json"
+
+on:
+  pull_request:
+    branches:
+      - "v*.*"
+      - "master"
+      - "feature/*"
+  push:
+    branches:
+      - "v*.*"
+      - "master"
+      - "feature/*"
+
+env:
+  PHP_VERSION: "8.2"
```

Comment:
Is there any particular reason to put 8.2 and not 8.3? (which would postpone the need to modify the version for the next required version upgrades).

Reply:
I essentially copied this from the "check arginfo" workflow. Depending on how the PHP version is determined in case it's absent, I'm also fine with not specifying a particular PHP version in this workflow.

Reply:
Any version works. So ok to delete.